scripts/rollup/build.js

```diff
@@ -57,7 +57,7 @@ function createBundle({entryPath, bundleType, destName}) {
 
   rollup({
     input: entryPath,
```

Comment:
Nice, Can you do it on `entryPath` variable, then?

Reply:
Hi, yes I can :)